gm4_mob_curing/data/gm4_mob_curing/functions/cure/liab/illusioner.mcfunction

```diff
@@ -0,0 +1,16 @@
+# Reverts the Illusioner to an Evoker
```

Comment:
Are Illusioners available in survival? I thought they were unused except maybe by mysterious midnights? Why would players want to convert an illusioner into an evoker if they could find one, for the expensive cost of a totem? Same goes for why would players want to convert an evoker into a pillager?